code/client/munkilib/CommandRepo.py

```diff
@@ -0,0 +1,311 @@
+<<<<<<< HEAD
```

Comment:
On Jun 6, 2016, at 5:55 PM, Clayton Burlison <notifications@github.com<mailto:notifications@github.com>> wrote:


In code/client/munkilib/CommandRepo.py<https://github.com/munki/munki/pull/616#discussion_r65995187>:

> @@ -0,0 +1,311 @@
> +<<<<<<< HEAD


You seem to have quite a few errors in this file from a git merge.

Weird. I didn’t get any messages about conflicts from the git command. I’ve pushed a fix for this.